main/config.ts

```diff
@@ -0,0 +1,10 @@
+/*
+  Direct usage of .env file would be preferred but
+  due to issue with Parcel CLI build which does not include/load the environment variables
+  I decided to use this way and have a simple way to define config.
+*/
+
+export const RPCH_SECRET_TOKEN = process.env.RPCH_SECRET_TOKEN || 'SECRET KEY' // TEST VALUE
+
+export const DISCOVERY_PLATFORM_API_ENDPOINT =
+  process.env.DISCOVERY_PLATFORM_API_ENDPOINT || 'https://discovery.rpch.tech'
```

Comment:
The default value would be set by the SDK if the parameter is not provided.
in order to avoid duplicating it here (should only be hold inside the SDK), it would be better to leave it empty if not set.
Then check on SDK instantiation if a value is set and if not just don't provide the option override to the SDK (so that the SDK takes it's own default value)

Reply:
I removed this redundant alternative.